programs/ssl/ssl_client2.c

```diff
@@ -444,16 +444,17 @@ static int ssl_sig_hashes_for_test[] = {
  * (Used in event-driven IO mode).
  */
 #if !defined(MBEDTLS_TIMING_C)
-void idle( mbedtls_net_context *fd,
+int idle( mbedtls_net_context *fd,
            int idle_reason )
 {
 #else
-void idle( mbedtls_net_context *fd,
+int idle( mbedtls_net_context *fd,
            mbedtls_timing_delay_context *timer,
            int idle_reason )
 {
 #endif
```

Comment:
Could you please move the brace outside the conditional compilation? As it is, the unbalanced braces screw up navigation in editors. Likewise in `ssl_server2.c`.

Reply:
Fixed in 9b2b66e.